src/actions/portfolio/handlers/bitcoin/index.js

```diff
@@ -0,0 +1,25 @@
+import { getInfoFromCoinGecko, getBtcFromSatoshis } from "../../../../utils";
+import Decimal from "decimal.js";
+
+export const getBitcoinPortfolio = async (
+    address,
+    fiatCurrency,
+    coinGeckoIds
+) => {
+    const response = await fetch(
+        `https://blockchain.info/rawaddr/${address}?limit=0`
```

Comment:
How many requests are permitted before hitting the rate-limiter?

Reply:
From the doc:

> Currently there is a limit of 1200 messages per minute. If the limit is exceeded, you will receive a rejected event. After waiting a minute, normal functionality will resume and you will be able to send messages again.